src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/beregning/TilsynBarnBeregningValideringUtil.kt

```diff
@@ -1,69 +1,23 @@
 package no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.beregning
 
-import no.nav.tilleggsstonader.kontrakter.felles.Datoperiode
-import no.nav.tilleggsstonader.kontrakter.felles.KopierPeriode
-import no.nav.tilleggsstonader.kontrakter.felles.Periode
 import no.nav.tilleggsstonader.kontrakter.felles.overlapper
 import no.nav.tilleggsstonader.sak.felles.domain.BarnId
 import no.nav.tilleggsstonader.sak.infrastruktur.exception.brukerfeilHvis
-import no.nav.tilleggsstonader.sak.infrastruktur.exception.brukerfeilHvisIkke
 import no.nav.tilleggsstonader.sak.infrastruktur.exception.feilHvis
 import no.nav.tilleggsstonader.sak.vedtak.TypeVedtak
-import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.beregning.TilsynBeregningUtil.brukPerioderFraOgMedRevurderFra
-import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.beregning.TilsynBeregningUtil.brukPerioderFraOgMedRevurderFraMåned
-import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.domain.Aktivitet
 import no.nav.tilleggsstonader.sak.vedtak.domain.Vedtaksperiode
-import java.time.LocalDate
 
 object TilsynBarnBeregningValideringUtil {
     fun validerPerioderForInnvilgelse(
         vedtaksperioder: List<Vedtaksperiode>,
-        aktiviteter: List<Aktivitet>,
         utgifter: Map<BarnId, List<UtgiftBeregning>>,
         typeVedtak: TypeVedtak,
-        revurderFra: LocalDate?,
     ) {
         if (typeVedtak == TypeVedtak.OPPHØR) {
             return
         }
         validerVedtaksperioder(vedtaksperioder)
-        validerAktiviteter(aktiviteter)
         validerUtgifter(utgifter)
-        validerOverlappendePeriodeOgUtgiftEtterRevurderFra(vedtaksperioder, utgifter, revurderFra)
-    }
-
-    private fun <P> validerOverlappendePeriodeOgUtgiftEtterRevurderFra(
```

Comment:
Er det litt tidlig å fjerne denne valideringen? 
Burde den finnes til at vi har tatt i bruk ny funksjonalitet i prod?

Reply:
Jeg la til valideringen av overlapp mellom vedtaksperiode og utgift nesten med en gang jeg begynt på denne oppgaven. https://github.com/navikt/tilleggsstonader-sak/commit/bdc2f95511e116693b0cd891eebfff2e7ddb90bb. Det skal ikke påvirke V1 koden, men var ment som en validering for V2. V1 koden reduserer vedtaksperioden (stønadsperioden) til å være like lang som utgiftsperioden automatisk ✨ 

Nå viser det seg at det ikke holder å validere overlapp mellom utgiftsperiode og vedtaksperiode. Vi må validere at utgiftsperioden inneholder vedtaksperioden. Det gjøres her:
https://github.com/navikt/tilleggsstonader-sak/pull/594/files#diff-e9feed4f719055743b6e451cce91e07c07f080ba5a3f63892cbe8e4fc50c133aR60-R89